nix/shell.nix

```diff
@@ -0,0 +1,13 @@
+{ pkgs, self', ... }:
+
+{
+  devShells.default = pkgs.mkShell {
+    packages = [
+      pkgs.nodejs
+      self'.packages.buildVscodeExtension
```

Comment:
why `self'`?

Reply:
It's one of the arguments flake parts gives you (https://flake.parts/module-arguments.html). Basically, all the flake outputs, qualified by system. So `self'.packages.buildVscodeExtension` is the same as `self.packages."${system}".buildVscodeExtension`.